pkg/files/files.go

```diff
@@ -119,27 +135,12 @@ func GetFiles(config config.Config) ([]string, error) {
 			return filePaths, err
 		}
 
-		_ = fs.WalkDir(os.DirFS(cwd), ".", func(path string, de fs.DirEntry, err error) error {
-			if err != nil {
-				return err
-			}
-
-			fi, err := de.Info()
-			if err != nil {
-				return err
-			}
-
-			if fi.Mode().IsRegular() {
-				filePaths = AddToFiles(filePaths, path, config)
-			}
-
-			return nil
-		})
+		return GetFilesFromDirectory(cwd, config)
```

Comment:
Here, it previously didn't return when the `git` command returned an error, but I think it was a mistake?